stylesheets/components/cards.css

```diff
@@ -0,0 +1,137 @@
+/* .cards-container{
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(18.75em, 1fr));
+    align-items: center;
+    justify-items: center;
+    gap: 1.25rem;
+} */
+
+.tr-card{
+    align-self: center;
+    padding: var(--space-sm);
+    border-radius: var(--space-xs);
+    background-color: var(--tr-background-lighter);
+    gap: var(--space-xs);
+    max-width: 16rem;
+    position: relative;
+    border: solid 1px var(--tr-gray);
+}
+
+.tr-card-banner img{
+    object-fit: cover;
+    border-radius: var(--space-xs) ;
+}
+
+.tr-card-header h2{
+    color: var(--tr-primary-lighter);
+}
+
+.tr-card-header h3{
+    font-size: 0.9rem;
+}
+
+.tr-card-body p{
+    font-size: .9rem;
+    color: var(--tr-text-secondary);
+}
+
+.pdt-price{
+    gap: ar(--space-sm);
+    align-items: center;
+}
+
+.curr-price{
+    font-size: var(--fs-md);
+    color: var(--tr-success);
+}
+
+.prev-price{
+    font-size: var(--fs-sm);
+    color: var(--tr-text-secondary);
+    text-decoration: line-through;
+}
+
+.tr-card-badge{
+    position: absolute;
+    font-size: .9rem;
+    top: 1.0rem;
+    left: 0;
+    padding: 5px var(--space-md);
+    background-color: var(--tr-accent-color) ;
+    color: var(--tr-text);
+}
```

Comment:
you can use badge component here.

Reply:
I haven't created the badge components yet, I will extract this in the badge.css once I create it